src/main/scala/nl.knaw.dans.easy.bagstore/command/CommandLineOptionsComponent.scala

```diff
@@ -118,9 +118,9 @@ trait CommandLineOptionsComponent {
 
     val stream = new Subcommand("stream") {
       descr("Retrieves an item by streaming it to the standard output")
-      val format: ScallopOption[ArchiveStreamType] = opt(name = "format",
+      val format: ScallopOption[ArchiveStreamType] = opt(name = "format", short = 'f',
         descr = "stream item packaged in this format (tar|zip)")
-      val forceInactive: ScallopOption[Boolean] = opt(name = "force-inactive",
+      val forceInactive: ScallopOption[Boolean] = opt(name = "force-inactive", //TODO no short since f is reserved for format?
```

Comment:
I would say you add `short = 'f'` here rather than to `format` to keep things consistent between all subcommands

don't forget to update the banner text above as well according to this...